ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift

```diff
@@ -204,12 +212,10 @@ class SourcePointClient: SourcePointProtocol {
             "messageId": messageId,
             "includeData": "{\"categories\": {\"type\": \"RecordString\"}}"
         ])! // swiftlint:disable:this force_unwrapping
```

Comment:
do we still need this `swiftlint:disable` ?

Reply:
@Nevazhnovu we do (check the `!` just behind the comment).